ComputerVision/SDK/DetectObjects/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Azure.CognitiveServices.Vision.ComputerVision;
+using Microsoft.Azure.CognitiveServices.Vision.ComputerVision.Models;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace DetectObjects
+{
+    class Program
+    {
+        private const string subscriptionKey = "0123456789abcdef0123456789ABCDEF"; //replace this with your key
+        private const string remoteImageUrl ="https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample4.png";
+
+        static void Main(string[] args)
+        {
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(subscriptionKey), 
+                new System.Net.Http.DelegatingHandler[] { });
+
+            // You must use the same region as you used to get your subscription
+            // keys. For example, if you got your subscription keys from westus,
+            // replace "westcentralus" with "westus".
+            // Free trial subscription keys are generated in the westcentralus
+            // region. If you use a free trial subscription key, you shouldn't
+            // need to change the region.
+            
+            computerVision.Endpoint = "https://westus.api.cognitive.microsoft.com";
+
+            // localImagePath = @"C:\Documents\LocalImage.jpg"
+            string localImagePath = Directory.GetCurrentDirectory() + @"../../../../../../Images\sample6.png";
+
+            Console.WriteLine("Images being analyzed ...");
+            var t1 = AnalyzeRemoteAsync(computerVision, remoteImageUrl);
+            var t2 = AnalyzeLocalAsync(computerVision, localImagePath);
+
+            Task.WhenAll(t1, t2).Wait(5000);
+            Console.WriteLine("Press ENTER to exit");
+            Console.ReadLine();
+        }
+
+        // Analyze a remote image
+        private static async Task AnalyzeRemoteAsync(ComputerVisionClient computerVision, string imageUrl)
```

Comment:
Let's call this DetectObjectsRemoteAsync, and the other case DetectObjectsLocalAsync